recipes/libyuv/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibyuvConan(ConanFile):
+    name = "libyuv"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://chromium.googlesource.com/libyuv/libyuv/"
+    description = "libyuv is an open source project that includes YUV scaling and conversion functionality."
+    topics = ["YUV", "libyuv", "google", "chromium"]
+    license = "BSD-3-Clause"
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "with_jpeg": [False, "libjpeg", "libjpeg-turbo"]}
```

Comment:
I'm not going to fight over it, but I would name the option `with_libjpeg`, in accordance with other recipes.

Reply:
![image](https://user-images.githubusercontent.com/14332136/99923127-43f9db80-2d6f-11eb-8445-883c65ede722.png)

japser use `with_libjpeg` and other recipes such as `zbar`, `opencv` use `with_jpeg`. I think both is ok.

Reply:
Yeah, looks like something our recipes don't agree about. Naming the options.

Reply:
#3614 option naming